client/src/App.js

```diff
@@ -11,6 +11,7 @@ import PasswordReset from "./pages/PasswordReset";
 import CreateCase from "./pages/CreateCase";
 import CreateAction from "./pages/CreateAction";
 import SinglePage from "./pages/SinglePage";
+import Overview from "./pages/Overview";
 /* =========================== Import Requirements */

```

Comment:
this could just be a blank space no?

Reply:
There would always be the searchbar, but I catch your drift. 

I am implementing a notification if the fetched data returns an empty array or is still loading.